base/channels.jl

```diff
@@ -102,6 +102,8 @@ end
 
 eltype{T}(::Type{Channel{T}}) = T
 
+capacity(c::Channel) = c.sz_max
```

Comment:
There's also `n_avail()` (so far used in the context of `Channel` only). To me `length()` is rather a synonym of `n_avail()`. `capacity()` could be potentially reused in other places to return what `sizehint!()` has set. But I would be glad to change it into anything core Julia developers decide.